src/main/java/com/criticalsoftware/UserRepository.java

```diff
@@ -0,0 +1,20 @@
+package com.criticalsoftware;
+
+import io.quarkus.mongodb.panache.PanacheMongoRepository;
+import io.quarkus.panache.common.Sort;
+import jakarta.enterprise.context.ApplicationScoped;
+import java.util.List;
+
+@ApplicationScoped
+public class UserRepository implements PanacheMongoRepository<User> {
+
+    // Este método procura um usuário pelo campo 'name'
```

Comment:
In code or comments, only english languaged is allowed.

Reply:
Translation done